packages/peregrine/src/hooks/useSortedImages.js

```diff
@@ -0,0 +1,13 @@
+import { useMemo } from 'react';
+
+export const useSortedImages = images => {
```

Comment:
We lost some self documenting naming here; since this hook does sort and filter. I agree with your idea that sorting should be passed as a prop to `useCarousel`, but I'm not sure that should even be the responsibility of the hook. This will probably be unique to the component, so I think filter/sorting should be done in component scope. Definitely would like to bounce that off you though.

Reply:
Yea, so with Toasts we did that. The store is just a map and the sorting is up to the component.

Let's talk tomorrow. Main points being:

* should we make carousel more generic and accept more than images
* where should sorting live? component vs hook?
* should we bake in default sorting and/or provide an API for passing a sorting function?

Reply:
@jimbo please help. Do we really want to make `useCarousel` and `useSortedImages` hooks? It feels like overkill and in the end the carousel hook is just a linked list. Perhaps `Carousel` is better off as a peregrine component?

Reply:
Ok I removed this hook but kept the `useCarousel`.